src/components/general/SideSheet/Standard/styles.less

```diff
@@ -0,0 +1,61 @@
+@import '~styles/mixins.less';
+@import '~styles/variables.less';
+
+.container {
+    box-sizing: border-box;
+    height: 100%;
+    border-left: 2px solid var(--color-black-alt4);
+    transition: width .1s;
+    overflow-y: auto;
+    overflow-x: hidden;
+    color: var(--color-primary-accent);
+    width: calc(var(--grid-unit)* 40px + 2px);
+    flex-shrink: 0;
+    background: var(--color-white);
+    justify-content: flex-end;
+
+    &.screenPlacementLeft{
+        border-left: none;
+        border-right: 2px solid var(--color-black-alt4);
+    }
+
+    @media (max-width: @mobile-max-width) {
+        width: 100%;
+        height: 100%;
+        left: 0;
+        top: 0;
+        bottom: 0;
+        right: 0;
+        transition: transform .1s ease-out;
+     
+        pointer-events: all;
+
+        /* &.isCollapsed {
+            
+        } */
+    }
+
+    .header{
+        display: flex;
+        align-items: center;
+        overflow: hidden;
+    }
+
+    .content {
```

Comment:
Maybe the css should reflect the html structure? .content ... { .header ... { .collapseButtonContainer ... }}}

Reply:
Fixed